examples/send_transactions/main.go

```diff
@@ -40,14 +40,14 @@ func demo() {
 	tx := flow.NewTransaction().
 		SetPayer(serviceAcctAddr).
 		SetProposalKey(serviceAcctAddr, serviceAcctKey.Index, serviceAcctKey.SequenceNumber).
-		SetScript([]byte(`transaction {
-
-  prepare(acc: &Account) {}
-
-  execute {
-    log("test")
-  }
-}`)).
+		SetScript([]byte(`
+			transaction {
+				prepare(signer: auth(Storage) &Account) {}
```

Comment:
not sure how to specify `auth(None)`

Reply:
Simply remove the authorization
```suggestion
				prepare(signer: &Account) {}
```

Reply:
oh yeah! thanks